### PR TITLE
Improve code and comment readibility in the met file scripts

### PR DESCRIPTION
**Pull request by Calebsakhtar**
This PR aims to improve code and comment readibility for the met file scripts. Specifically, the following was done:

- Improved the instructions on how to run the script to the met readme
- Added units to some relevant code variable names to help with readability
- Improved the code comments and fixed some mistakes in them
- Specified the imported functions from the src modules

### CODE REVIEW

**Comment by Calebsakhtar** [comment]
@sdeastham Would you mind giving ths a quick review?

File: scripts/simple_met_input/README.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 This folder contains python scripts to produce idealised met files containing trapezoidal moist layers (in an altitude
 vs RHi plot).
 
+TO RUN THE "main.py" SCRIPT, PLEASE OPEN THE FOLDER "SIMPLE_MET_INPUT" AS YOUR IN YOUR EDITOR/SET IT AS YOUR PYTHON RUNDIR.
```
Comment (sdeastham) [inline scripts/simple_met_input/README.txt]: Grammatical error here - "as your in your"?

Comment (Calebsakhtar) [reply scripts/simple_met_input/README.txt]: @sdeastham Fixed in commit https://github.com/MIT-LAE/APCEMM/pull/34/commits/33b3b6ca39e6ef0d8be220a6554c1b904a3a1190

**Comment by sdeastham** [comment]
Verified that the scripts work, the output meteorology looks reasonable and runs with APCEMM, and the comment changes are an improvement. Thanks @Calebsakhtar !

**Pull request merged**